saleor/product/migrations/0126_product_default_variant.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 3.1.1 on 2020-09-22 11:49
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+def set_default_variants(apps, schema_editor):
```

Comment:
Pay special attention to this while reviewing, this is done by the most bulletproof way which we discussed yesterday with the team.

Reply:
Looks good IMO